planning/behavior_path_planner/autoware_behavior_path_start_planner_module/README.md

```diff
@@ -469,17 +469,18 @@ Pull out distance is calculated by the speed, lateral deviation, and the lateral
+| maximum_lateral_jerk                           | [m/s3] | double | maximum lateral jerk                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | 2.0           |
+| minimum_lateral_jerk                           | [m/s3] | double | minimum lateral jerk                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | 0.1           |
+| minimum_shift_pull_out_distance                | [m]    | double | minimum shift pull out distance. if calculated pull out distance is shorter than this, use this for path generation.                                                                                                                                                                                                                                                                                                                                                                   | 0.0           |
+| maximum_curvature                              | [1/m]  | double | maximum curvature. The pull out distance is calculated so that the curvature is smaller than this value.                                                                                                                                                                                                                                                                                                                                                                               | 0.07          |
+| end_pose_curvature_threshold                   | [1/m]  | double | The curvature threshold which is used for calculating the longitudinal distance of shift. The shift end pose is shifted forward so that its curvature is less than this value. This is to prevent the generated path from having a large curvature when the end pose is on a curve. If a point with a curvature below the threshold is not found, the longitudinal distance is used as the distance to the point with the lowest curvature among the points beyond a certain distance. | 0.01          |
```

Comment:
```suggestion
| end_pose_curvature_threshold                   | [1/m]  | double | The curvature threshold which is used for calculating the longitudinal distance of shift. The shift end pose is shifted forward so that curvature on shift end pose is less than this value. This is to prevent the generated path from having a large curvature when the end pose is on a curve. If a shift end pose with a curvature below the threshold is not found, the longitudinal distance is used as the distance to the point with the lowest curvature among the points beyond a certain distance. | 0.01          |
```
Btw, `longitudinal distance` is different from `shift pull out distance`?

Reply:
it's same!

Reply:
I will update the docs

Reply:
updated